flagship/src/visitor/DefaultStrategy.ts

```diff
@@ -634,7 +659,9 @@ export class DefaultStrategy extends VisitorStrategyAbstract {
       return
     }
 
-    return this.sendActivate(flag, functionName)
+    await this.sendActivate(flag, functionName)
+
+    this.onUserExposedCallback({ flag, visitor: this.visitor })
```

Comment:
Why not put that in the `sendActivate` function, so that the callback is called anytime the activation is done. Unless we want the callback to be called only for `userExposed` function, and not the `activateModification` for instance ?